daffodil-io/src/test/scala/org/apache/daffodil/io/TestDirectOrBufferedDataOutputStream.scala

```diff
@@ -113,7 +113,7 @@ class TestDirectOrBufferedDataOutputStream {
 
   @Test def testCollapsingTwoBuffersIntoDirect {
 
-    val baos = new ByteArrayOrFileOutputStream(2000 * (1 << 20), Paths.get("."), Maybe.Nope)
+    val baos = new ByteArrayOrFileOutputStream(2000 * (1 << 20), new File("."), Maybe.Nope)
     val layered = newDirectOrBufferedDataOutputStream(baos, null)
 
     val hw = "Hello World!"
```

Comment:
Do we need some new TDML or unit tests that validate that these new tunables work and switching to a file works correctly?  I dont think any of these tests have small enough limits to ever switch to a file, and since this is tunable based we can actually have tests for it, unlike the other 2GB changes. A good starting point would be to look at the code coverage and make sure all the new paths are hit.

Reply:
You are correct, I have tested them by manually changing the tunable.  I forgot that with the tunable we can actually set up some TDML test cases.  Will do.